### DISCUSSION TITLE
射撃武器につけるスレイングの方針

### DISCUSSION THREAD

**Discussion by dis-** [hengband/hengband#250] [Ideas]
弓にスレイが乗るようになったので、色々と実装できるようになった。
バランスを破壊しない程度に色々な種類の弓を与えてプレイングに幅を出したい。
全体的な方針の叩き台を作ってみる。

・下級スレイ、元素エゴは倍率をマイルドに(1.5倍が目安)。エゴや☆にも気軽に使えるスペックにする。
・上級スレイは高威力に(2.5～3.0倍が目安)。出にくくしてプレミア感を与えたい。
・対邪の弓は最終火力に影響するので取扱は慎重にする。★に付けるなら威力の調整必須。
・理力は威力じゃなくて命中を盛ってクリティカル率を上げると楽しそう

**Comment by ghost** [comment]
ここで聞くべき話題か曖昧なのですが、弓のスレイと矢玉のスレイの関係はどうなるのでしょうか？
(弓にスレイがつくなら矢玉のスレイは廃止される方針とかでしょうか？？)

**Reply by iks3** [reply]
矢/弾のスレイングについては、廃止されないと思います。
弓にスレイをたくさんばらまく予定ではありませんので、矢/弾のスレイは使えると思います。

**Reply by ghost** [reply]
ありがとうございます、矢玉スレイも残るのですね。
恥ずかしながらコードが分からないのでスポイラーの説明を読んでみましたが、
> xdyとは、y面ダイスをx個振った合計値である。期待値はx*(1+y)/2。
> 矢のダイスを振り、これをベースとする。
> 中略
> ダイス目にダメージを加え、弓の倍率がかかった結果をn倍にする。

バルド弓/矢や理力弓など例外を除くとスレイ弓でエゴ矢を撃つと大きい方のスレイ倍率だけ掛かるという理解で良いでしょうか？
例えば
　「★ジャイアント・キリング(滅ジャイアント2.0倍)」で「冷気の鉄弾(冷気1.7倍)」を「炎の巨人スルト」
に撃った場合、スレイ倍率は滅ジャイアントの2.0倍だけが乗るという事でしょうか？

**Reply by dis-** [reply]
現在の実装においてはそうなっています。今後の変更・修正の予定は今のところはありません。
質問コーナーを作りましたので、以後そちらでお願いします。#356

**Reply by ghost** [reply]
お手数をお掛けします、今後何か不明点出ましたらそちらに質問投げさせていただきます。
